### PR TITLE
feature/8889-ActiveClaimsAlertAndLinkAndDetailsAndPanel

### PR DESCRIPTION
**Pull request by Sparowhawk**
## Description of Change
VQA'd in slack thread, added active alert for file requests/submit evidence button(this is behind feature flag)
Added what you claimed section to the status tab(behind feature flag) spacing for it will be fixed on VABulletList spacing bug ticket

Changed links on active claims at bottom to the link with analytics component(not behind feature flag)

Updated copy for why we combine panel(not behind feature flag)

## Screenshots/Video

Screenshot 2024-07-23 at 4 05 00 PM
Screenshot 2024-07-23 at 4 05 05 PM
Screenshot 2024-07-23 at 4 05 10 PM
Screenshot 2024-07-23 at 4 05 18 PM

## Testing
yarn test/detox updated

- [x] Tested on iOS 
- [x] Tested on Android 

## Reviewer Validations
Active Claim: Show a conditional primary action:

 For open File Requests, show blue informational Alert with [Review file requests] primary button – use the Design System Alert Component. For no open file requests, show [Submit evidence] primary button (button can be shown as long as claim.attributes.open is true, see [#8864](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8864) for details).
[Figma mockup](https://www.figma.com/design/kSlXLwTVLunM0fqCG95k5x/%F0%9F%94%8D-Claims-2.0---Working---VA-Mobile?node-id=1586-1774&t=oNi2ZnGJTD2vzqXa-4)
Link copy reads: Find out why we sometimes combine claims (see [copy doc for Large Panel copy](https://docs.google.com/spreadsheets/d/16waC665d7uNRQmM83SJIm9bTDyRePOuB6r9G_aQE8Tw/edit?gid=831099110#gid=831099110&range=B2) for content inside panel)
 Content aligns with [copy doc](https://docs.google.com/spreadsheets/d/16waC665d7uNRQmM83SJIm9bTDyRePOuB6r9G_aQE8Tw/edit?gid=728579729#gid=728579729)

 Include section: What you've claimed (bulleted list from current Details segment)

 VQA with Lauren / Misty
## PR Checklist

  **Reviewer:** Confirm the items below as you review

- [x] PR is connected to issue(s)
- [x] Tests are included to cover this change (when possible)
- [x] No magic strings (All string unions follow the [Union -> Constant](https://github.com/department-of-veterans-affairs/va-mobile-app/blob/develop/VAMobile/src/constants/common.ts) type pattern)
- [x] No secrets or API keys are checked in
- [x] All imports are absolute (no relative imports)
- [x] New functions and Redux work have proper TSDoc annotations

## For QA

[Run a build for this branch](https://github.com/department-of-veterans-affairs/va-mobile-app/actions/workflows/on_demand_build.yml)

### CODE REVIEW

File: VAMobile/src/screens/BenefitsScreen/ClaimsScreen/ClaimDetailsScreen/ClaimStatus/ClaimTimeline/ClaimTimeline.tsx [hunk 1]
```diff
@@ -35,12 +36,12 @@ function ClaimTimeline({ attributes, claimID }: ClaimTimelineProps) {
 
   return (
     <Box>
-      {itemsNeededFromVet && !attributes.waiverSubmitted && (
+      {itemsNeededFromVet && !attributes.waiverSubmitted && !featureEnabled('claimPhaseExpansion') && (
```
Comment (dumathane) [inline VAMobile/src/screens/BenefitsScreen/ClaimsScreen/ClaimDetailsScreen/ClaimStatus/ClaimTimeline/ClaimTimeline.tsx:39]: I don't love this code. This is hard to follow logically. Essentially we are...changing nothing on the screen here but updating some content string values, but also removing this alertbox in the version of the code where the feature flag is on. We should have a deprecatedClaimTimeline file here that we navigate to when the feature flag is off, and the copy shouldn't have this alertbox.

Comment (Sparowhawk) [reply VAMobile/src/screens/BenefitsScreen/ClaimsScreen/ClaimDetailsScreen/ClaimStatus/ClaimTimeline/ClaimTimeline.tsx]: updated

File: VAMobile/src/screens/BenefitsScreen/ClaimsScreen/ClaimDetailsScreen/ClaimDetailsScreen.test.tsx [hunk 2]
```diff
@@ -53,13 +67,13 @@ context('ClaimDetailsScreen', () => {
   describe('when the selected tab is status', () => {
     it('should display the ClaimStatus component', async () => {
       when(api.get as jest.Mock)
```
[the remark this answers is not in the document]
Comment (Sparowhawk) [reply VAMobile/src/screens/BenefitsScreen/ClaimsScreen/ClaimDetailsScreen/ClaimDetailsScreen.test.tsx:69]: updated

File: VAMobile/src/screens/BenefitsScreen/ClaimsScreen/ClaimDetailsScreen/ClaimStatus/ClaimStatus.tsx [hunk 3]
```diff
@@ -49,43 +49,32 @@ function ClaimStatus({ claim, claimType }: ClaimStatusProps) {
...
-          testId: a11yLabelVA(t('claimDetails.whatShouldIDoIfDisagree')),
-        },
-      ]
-
       // TODO: determine when showCovidMessage prop for EstimatedDecisionDate would be false
 
       return (
         <Box>
-          {claim && <ClaimTimeline attributes={claim.attributes} claimID={claim.id} />}
+          {claim && featureEnabled('claimPhaseExpansion') && (
```
[the remark this answers is not in the document]
Comment (Sparowhawk) [reply VAMobile/src/screens/BenefitsScreen/ClaimsScreen/ClaimDetailsScreen/ClaimStatus/ClaimStatus.tsx:57]: I could honestly think at this point it doesn't really matter cause all it would do is shift around where in the files we would have this check and maybe save us a few minutes when cleaning up. I think in the future as long as we have the feature flag/release ASAP stuff outlined before hand from the beginning we can architect this much better throughout the process.

File: VAMobile/src/screens/BenefitsScreen/ClaimsScreen/ClaimDetailsScreen/ClaimDetailsScreen.tsx [hunk 4]
```diff
@@ -139,6 +153,11 @@ function ClaimDetailsScreen({ navigation, route }: ClaimDetailsScreenProps) {
     navigateTo('ClaimLettersScreen')
   }
 
+  const fileRequestsPress = () => {
+    logAnalyticsEvent(Events.vama_claim_review(claimID, attributes.claimType, count))
```
[the remark this answers is not in the document]
Comment (Sparowhawk) [reply VAMobile/src/screens/BenefitsScreen/ClaimsScreen/ClaimDetailsScreen/ClaimDetailsScreen.tsx:157]: she created separate tickets for the analytics so let's just address them there and not bloat this up too much

File: VAMobile/src/screens/BenefitsScreen/ClaimsScreen/ClaimDetailsScreen/ClaimDetailsScreen.tsx [hunk 5]
```diff
@@ -173,19 +192,56 @@ function ClaimDetailsScreen({ navigation, route }: ClaimDetailsScreenProps) {
         </Box>
       )
     }
+    if (claimPhaseExpansionFlag) {
+      if (count > 0) {
+        const buttonProps: ButtonProps = {
```
[the remark this answers is not in the document]
Comment (Sparowhawk) [reply VAMobile/src/screens/BenefitsScreen/ClaimsScreen/ClaimDetailsScreen/ClaimDetailsScreen.tsx:197]: she created separate tickets for the analytics so let's just address them there and not bloat this up too much

**Review by alexandec — APPROVED** [review]

**Review by alexandec — DISMISSED** [review]

**Review by dumathane — DISMISSED** [review]

**Review by timwright12 — APPROVED** [review]

**Review by rbontrager — APPROVED** [review]

**Review by dumathane — APPROVED** [review]
